CHANGELOG.md

```diff
@@ -1,6 +1,186 @@
 # Change Log
 ## [Releases](https://github.com/NetApp/harvest/releases)
 
+## 25.05.0 / 2025-05-15 Release
+pushpin: Highlights of this major release include:
+## :star: New Features
+
```

Comment:
Maybe add this one lower down in the list
- Harvest includes a beta template to collect ethernet switch ports. Thanks to Robert Watson for reporting!